identity-concentrator/outputs.tf

```diff
@@ -1,3 +1,16 @@
+#
+# Service Account Outputs
+#
+
```

Comment:
nit, lookup some sample code online, seems we don't need this extra blank line?

Reply:
Sure, will fix